build/build.go

```diff
@@ -255,7 +255,7 @@ func toSolveOpt(ctx context.Context, node builder.Node, multiDriver bool, opt Op
 	supportsAttestations := bopts.LLBCaps.Contains(apicaps.CapID("exporter.image.attestations")) && nodeDriver.Features(ctx)[driver.MultiPlatform]
 	if len(attests) > 0 {
 		if !supportsAttestations {
-			return nil, nil, errors.Errorf("attestations are not supported by the current buildkitd")
+			return nil, nil, errors.Errorf("attestations are not supported by the current BuildKit daemon")
```

Comment:
Can we make this error better and say that containerd storage needs to be enabled for `!multiPlatform`. I've seen people confused by this. Basically different error messages for the case where attestations are not supported and when multi-platform is not supported.